DOCSP-48487 Move usage examples (#1071)
mongodb/docs-node: source/crud/delete.txt

more documents. These methods accept a query document that matches the
documents you want to delete.

.. note::

>   If your application requires the deleted document after deletion,
>   consider using the
>   `collection.findOneAndDelete() <{+api+}/classes/Collection.html#findOneAndDelete>`__
>   method, which has a similar interface to ``deleteOne()`` but also
>   returns the deleted document.

Reviewer: ```suggestion
  If your application uses information about the document after deletion,
  you can use the
  `collection.findOneAndDelete() <{+api+}/classes/Collection.html#findOneAndDelete>`__
  method, which has a similar interface to ``deleteOne()`` but also
  returns the deleted document.
```

If the delete operation is successful, these statements print the number of documents
deleted by the associated operation.

.. _node-usage-deleteone:

deleteOne() Usage Example
~~~~~~~~~~~~~~~~~~~~~~~~~

.. include:: /includes/crud/example-intro.rst

The following code is a complete, standalone file that performs a delete one
operation:

.. tabs::

  .. tab:: JavaScript
     :tabid: javascript

     .. literalinclude:: /code-snippets/usage-examples/deleteOne.js
        :language: javascript
        :linenos:

  .. tab:: TypeScript
     :tabid: typescript

     .. literalinclude:: /code-snippets/usage-examples/deleteOne.js
        :language: javascript
        :linenos:

.. note:: Identical Code Snippets

   The JavaScript and TypeScript code snippets above are identical. There are no

>    TypeScript specific features of the driver relevant to this use case.

Reviewer: S: since this is reused often, consider moving to an include

Author: Moved to an include, reworded, and removed tabs as per Rachel's suggestion above.

Running the preceding example, you see the following output:

.. code-block:: none
   :copyable: false

   Successfully deleted one document.

If you run the example more than once, you see the following output because
you deleted the matching document in the first run:

.. code-block:: none
   :copyable: false

   No documents matched the query. Deleted 0 documents.

.. _node-usage-deletemany:

deleteMany() Usage Example
~~~~~~~~~~~~~~~~~~~~~~~~~~

.. include:: /includes/crud/example-intro.rst

The following code is a complete, standalone file that performs a delete many
operation:

.. tabs::

   .. tab:: JavaScript
      :tabid: javascript

      .. literalinclude:: /code-snippets/usage-examples/deleteMany.js
         :language: javascript
         :linenos:

   .. tab:: TypeScript
      :tabid: typescript

      .. literalinclude:: /code-snippets/usage-examples/deleteMany.ts
         :language: typescript
         :linenos:

Running the preceding example for the first time, you see the following output:

.. code-block:: none
   :copyable: false

   Deleted 19 documents

> If you run the example more than once, you see the following output because
> you deleted the matching documents in the first run:
> 

Reviewer: ```suggestion
If you run the example more than once, the code produces following output because
you deleted the matching documents in the first run:

```
